src/SignalRServiceExtension/TriggerBindings/ServerlessHub.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Management;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    /// <summary>
+    /// When a class derived from <see cref="ServerlessHub"/>,
+    /// all the method in the are identified as using class based model.
+    /// </summary>
+    public abstract class ServerlessHub : IDisposable
+    {
+        private bool _disposed;
+
+        public ServerlessHub()
+        {
+            var hubName = GetType().Name;
```

Comment:
>hubName [](start = 16, length = 7)

add HubName to public immutable property? So that in functions, negotiation method could get the value as well?

Reply:
They are in the different pipelines. The class instance constructs every time the function is invoked. So, the properties of the class cannot be used in attribute parameters. If the negotiation method need to get hub name, it also needs to use `MethodInfo` in binding.